lib/manager/cocoapods/artifacts.ts

```diff
@@ -67,11 +67,14 @@ export async function updateArtifacts({
   );
   const tagConstraint = match?.groups?.cocoapodsVersion ?? null;
 
+  const cacheDir = await ensureCacheDir('./others/cocoapods', 'CP_HOME_DIR');
+  logger.debug(`Using cocoapods home ${cacheDir}`);
```

Comment:
better use trace here?

Reply:
Agree. I want to move logger inside `ensureCacheDir` in subsequent PR

Reply:
Let's keep it for now